### PR TITLE
feat: initialized project

### PR DESCRIPTION
**Pull request by antonio-marasescu**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+services:
+  fabula-db:
+    image: 'mongo'
+    container_name: 'fabula-db'
+    environment:
+      - MONGO_INITDB_DATABASE=fabuladb
+      - MONGO_INITDB_ROOT_USERNAME=fabularootuser
+      - MONGO_INITDB_ROOT_PASSWORD=fabularootpass
+    ports:
+      - '27017-27019:27017-27019'
```
Comment (paulcosma97) [inline docker/docker-compose.yml:12]: Why multiple ports?

File: docker/mongo/mongo-db-init.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+db.createUser({
```
Comment (paulcosma97) [inline docker/mongo/mongo-db-init.js:1]: We should move this to a `migrations` dir

**Review by paulcosma97 — COMMENTED** [review]
Maybe move all of this inside a `server` dir so we can have the Unity project at the root level too

File: docker/docker-compose.yml (hunk 1 above)
Comment (antonio-marasescu) [reply docker/docker-compose.yml:12]: just took an old image :))) just 27017 then

File: docker/mongo/mongo-db-init.js (hunk 2 above)
Comment (antonio-marasescu) [reply docker/mongo/mongo-db-init.js:1]: ye

**Comment by antonio-marasescu** [comment]
> Maybe move all of this inside a `server` dir so we can have the Unity project at the root level too

So we will go for a monorepo then? I was thinking of keeping the unity game in a separate project, but sure

File: docker/docker-compose.yml (hunk 1 above)
Comment (antonio-marasescu) [reply docker/docker-compose.yml]: done

File: docker/mongo/mongo-db-init.js (hunk 2 above)
Comment (antonio-marasescu) [reply docker/mongo/mongo-db-init.js]: done

Comment (paulcosma97) [reply docker/mongo/mongo-db-init.js]: Now t

> > Maybe move all of this inside a `server` dir so we can have the Unity project at the root level too
> 
> So we will go for a monorepo then? I was thinking of keeping the unity game in a separate project, but sure

Yup

**Pull request merged**